### ISSUE TITLE
Post-deployment event scripts

### ISSUE THREAD

**Issue by RemiSlv** [mkloubert/vscode-deploy-reloaded#73]
### Description

Hi there, 
More of a feature request here:
It would be nice to have a way of running post-deployment scripts.
The `uploaded` option from targets is already helpful for that but limited in my case:

### Actual behavior

Using the `uploaded` option in targets runs the corresponding script for each processed file, as expected.

### Expected behavior

I'm after a way of running a script only once my deployment is completed.

Example:
I am deploying code via sftp and would like to run a compilation command after all my files have been deployed.

Is there already a way to achieve what I'm trying to do ?
I also noticed the `commands` setting but there doesn't seem to be a way of automatically running them as a post-deployment action.

### Your environment

- Operating system: Windows 10
- Visual Studio Code version: 1.24.0
- Extension version: 0.75.0

**Comment by RemiSlv** [comment]
Missed it in the wiki. It's already there indeed:
https://github.com/mkloubert/vscode-deploy-reloaded/wiki/target_sftp#commands-

You can close this issue. Thanks!

**Comment by Rutek** [comment]
Are you sure this reply is still up to date? I'm reading referenced wiki page for the third time and I can't see any possibility to execute command AFTER all files are uploaded (once per deployment, not for every file).

**Comment by mkloubert** [comment]
@Rutek 

What is your scenario? What do you expect?

**Comment by Rutek** [comment]
@mkloubert I would like to run Composer once after upload all the files. It has to be run by SSH(SFTP).
